src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/DeleteOrganisationTest.java

```diff
@@ -63,12 +63,6 @@ public void ac3_error_when_delete_an_organisation_with_unknown_org_identifier_sh
         professionalApiClient.deleteOrganisation(orgIdentifier, hmctsAdmin, HttpStatus.NOT_FOUND);
     }
 
-    @Test
-    @ToggleEnable(mapKey = mapKey, withFeature = true)
-    public void ac4_error_when_delete_an_organisation_with_invalid_org_id_should_return_400() {
```

Comment:
not getting why we deleted? @ToggleEnable(mapKey = mapKey, withFeature = false). is it not like that? Are we changing delete one?

Reply:
We have decided to remove 400 bad request related test cases from functional level

Reply:
ok ok my bad...I though 403.